lib/Driver.js

```diff
@@ -139,9 +138,21 @@ module.exports = class Driver {
    *
    * @returns {string} Returns the key's id.
    */
-  async computeKeyId({key}) {
+  computeKeyId({key}) {
     return `did:key:${key.fingerprint()}#${key.fingerprint()}`;
   }
+
+  /**
+   * Generates a public verification method from a keypair.
+   *
+   * @param {LDKeyPair} key
+   *
+   * @returns {string} Returns the key's id.
+   */
+  getPublicVerificationMethod({key}) {
+    return (({id, type, controller, publicKeyBase58}) =>
+      ({id, type, controller, publicKeyBase58}))(key);
```

Comment:
I'm not really sure what's supposed to be going on here. What is this method supposed to do? It doesn't really match my expectations given the documentation and method name.

Reply:
The method takes in a keypair and returns an object containing only the public key information. It was named "getPublicVerificationMethod" for this reason. Alternate names: "getVerificationMethod()", "getPublicKey()".

Reply:
Changed to _verificationMethodFromKeyPair() in 4147f8100fc85fbc8db6158633cea84bf24d746c.